org.lflang/src/org/lflang/TargetConfig.java

```diff
@@ -265,6 +266,9 @@ public class TargetConfig {
     /** Rust-specific configuration. */
     public final RustTargetConfig rust = new RustTargetConfig();
 
+    /** Path to a C file used by the Python target to setup federated execution. */
+    public String fedSetupPreamble = null;
```

Comment:
Target specific members should be added only in derived classes.

Reply:
Sounds reasonable, but there is some other target-specific stuff here already (i.e., the `rust` member right above). I don't understand why it's here at all, or what it is, actually. The comment also seems wrong.

Reply:
https://issue.lf-lang.org/1558